ps_categorytree.php

```diff
@@ -177,13 +178,26 @@ public function getTree($resultParents, $resultIds, $maxDepth, $id_category = nu
             $link = $name = $desc = '';
         }
 
-        return [
+        $return = [
             'id' => $id_category,
             'link' => $link,
             'name' => $name,
             'desc' => $desc,
             'children' => $children,
         ];
+
+        if(Configuration::get('BLOCK_CATEG_SHOW_NUMB_PRODS') == 1){
```

Comment:
Does not hesitate to use the coding standards and the fixer as it : https://devdocs.prestashop.com/1.7/development/coding-standards/#making-your-code-follow-our-coding-standards